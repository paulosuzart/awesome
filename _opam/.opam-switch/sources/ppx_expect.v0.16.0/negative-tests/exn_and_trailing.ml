let%expect_test _ =
  print_endline "hello";
  if true then raise Exit;
  [%expect {| hello |}]
[@@expect.uncaught_exn {| Exit |}]
;;
