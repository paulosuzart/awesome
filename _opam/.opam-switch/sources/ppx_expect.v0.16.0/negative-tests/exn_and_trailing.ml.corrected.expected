let%expect_test _ =
  print_endline "hello";
  if true then raise Exit;
  [%expect.unreachable]
[@@expect.uncaught_exn {|
  Exit
  Trailing output
  ---------------
  hello |}]
;;
